src/infection-model/seir.tsx

```diff
@@ -159,6 +203,23 @@ function getCurveProjections(inputs: SimulationInputs & CurveProjectionInputs) {
   ageGroupFatalityRates[ageGroupIndex.age85] = 0.1885;
   ageGroupFatalityRates[ageGroupIndex.staff] = 0.026;
 
+  // calculate R0 adjusted for housing type and capacity
+  let rateOfSpreadCells = R0Cells[rateOfSpreadFactor];
+  const rateOfSpreadCellsAdjustment = 0.8; // magic constant
+  rateOfSpreadCells =
+    rateOfSpreadCells -
+    (1 - facilityOccupancyPct) *
+      (rateOfSpreadCells - rateOfSpreadCellsAdjustment);
+  let rateOfSpreadDorms = R0Dorms[rateOfSpreadFactor];
+  const rateOfSpreadDormsAdjustment = 1.7; // magic constant
+  rateOfSpreadDorms =
+    rateOfSpreadDorms -
+    (1 - facilityOccupancyPct) *
+      (rateOfSpreadDorms - rateOfSpreadDormsAdjustment);
+
+  const ratioExposedToInfected = 0.487804878;
```

Comment:
Can we comment about why this variable is set to this value?

Reply:
This is `dIncubation`/`dInfectious` and it's the ratio of total exposed to total infected pop.